data/studentApplyData.js

```diff
@@ -195,26 +262,30 @@ const studentProcess = [
   //   icon: <Interview />,
   //   date: socialRoundDate
   // },
-  {
-    title: "Take Home Exercise",
-    detail:
-      "A small, take home project designed to expose you to the tasks and technologies you will encounter while working with your project teams. More instructions will be provided then.",
-    icon: <Contract />,
-    date: takeHomeExerciseDate
-  },
-  {
-    title: "Interviews",
-    detail:
-      "Hack4Impact looks for students who are hungry to learn, excellent teammates, and are passionate for tech and social impact. Our interviews will be 1 hour long and you will get meet a couple of our members.",
-    icon: <Interview />,
-    date: firstRoundInterviewDate
-  }
+  // {
+  //   title: "Take Home Exercise",
+  //   detail:
+  //     "A small, take home project designed to expose you to the tasks and technologies you will encounter while working with your project teams. More instructions will be provided then.",
+  //   icon: <Contract />,
+  //   date: takeHomeExerciseDate
+  // },
+  // {
+  //   title: "Interviews",
+  //   detail:
+  //     "Hack4Impact looks for students who are hungry to learn, excellent teammates, and are passionate for tech and social impact. Our interviews will be 1 hour long and you will get meet a couple of our members.",
+  //   icon: <Interview />,
+  //   date: firstRoundInterviewDate
+  // }
 ];
 
 const closedText = (
   <p>
-    <b><em>Our current application period for {currentSemester} is currently open.</em></b> We are <b><em>only accepting Product Designers</em></b> for the Spring semester.
-    For those that are interested in software development, stop by our website in the future to view updated information about the next application period!
+    Our application period for {currentSemester} <b><em>will open soon. </em></b> We will be accepting 
+    <b> Academy Members, Software Developers and a Product Designer</b> for the
```

Comment:
we shouldn't be saying anything that implies how many people we will be accepting

Reply:
got it! updated the wording in that instance, on the Notion page and interest form